.github/workflows/cmake.yml

```diff
@@ -203,6 +203,10 @@ jobs:
       # This is so that device binaries can find the sycl runtime library
       run: cmake --build ${{github.workspace}}/build -j $(nproc)
 
+    - name: Get information about platform
+      if: always()
+      run: .github/scripts/get_system_info.sh
```

Comment:
you could additionally run (from built repo) `(repo)/build/bin/urinfo || true`

Reply:
done in script